dcos/dcos.go

```diff
@@ -0,0 +1,189 @@
+package dcos
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/libopenstorage/secrets"
+	api "github.com/portworx/dcos-secrets"
+)
+
+// Keys for the config to initialize the DC/OS secrets client
+const (
+	KeyUsername   = "username"
+	KeyPassword   = "password"
+	KeyCACertFile = "cacert"
+	KeyDcosURL    = "dcos_url"
+)
+
+const (
+	Name           = "dcos"
+	SecretsAuthor  = "osd"
+	KeySecretStore = "secret_store"
```

Comment:
I don't think any of these need to be exported outside the package right?

Reply:
The Name is used in px.
PX will also use the KeySecretStore to set the value. Right now it does not set because it is default.

Author is not used outside. I just confirmed with DCOS folks, they are going to keep only the value field in the secret. Will remove it from here now and later from the dcos-secrets repo.